website/src/data/navigation.yaml

```diff
@@ -62,6 +62,8 @@
       path: /deployment-examples/surge
     - title: Deploying to Cloudflare Pages
       path: /deployment-examples/cloudflare
+    - title: Deploying to Google Cloud
```

Comment:
```suggestion
    - title: Deploying to Google Cloud with Cloud Storage
```

Reply:
Thank you! updated here: https://github.com/commercetools/merchant-center-application-kit/pull/2740/commits/e5c83a6c761487a55d808cc01444b246e529e4f0